### DISCUSSION TITLE
Can I use a Copilot for business subscription (granted by my company) with my Github account?

### DISCUSSION THREAD

**Discussion by allenrobel** [community/community#64643] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hi,

I applied for (within my company), and was granted, a subscription to Github Copilot for business.  I have a personal Github account (which I use in association with my company).  I received the confirmation email at my company email address, with subject "You have been granted access to GitHub Copilot".

I've installed Copilot in VS Code.  How do I tie this Copilot for business subscription to my Github account (it's a personal account), or is this not possible?  Or, is this not even needed (e.g. if Copilot is not associated with Github accounts for billing purposes).

I guess I'm a bit confused how payments are going to work in my case.

Thanks!

Allen

**Comment by Franklyn-R-Silva** [comment]
Hi Allen,

It's great that you've been granted a GitHub Copilot for Business subscription within your company. Regarding your question about tying the Copilot for business subscription to your personal GitHub account, it's important to note that GitHub Copilot for Business is typically associated with your company's billing rather than individual GitHub accounts.

Since you received the confirmation email at your company email address and the subscription is intended for business use, the billing and subscription management would likely be handled by your company's GitHub organization. The subscription is typically linked to the organization's billing details and wouldn't require direct linkage to individual accounts within the organization.

Your personal GitHub account can still be used for accessing and using GitHub Copilot within your work projects, but the billing and subscription aspects are typically managed at the organization level.

If you have specific questions about how billing will work or if there are any special arrangements for your case, it's recommended to reach out to GitHub support or your company's IT department. They should be able to provide you with accurate information about the billing setup and how it relates to your personal GitHub account.

I hope this clarifies things for you. If you have further questions, don't hesitate to ask.

Best regards,

**Comment by allenrobel** [comment]
Thank you Franklyn for the timely (and excellent) reply!  I ended up finding some internal resources that explain how this works for our company (entailed logging into Github under a newly-generated corporate account before installing Copilot in VS Code).  I think I'm good now.
